src/dom/map-setup.ts

```diff
@@ -78,10 +57,8 @@ export function animateLap(lapJSON: ILapDataset) {
     let long = lapJSON.dataSet[counter]["Lon."] * 0.000001;
     let lat = lapJSON.dataSet[counter]["Lat."] * 0.000001;
 
```

Comment:
Did you perform safely checks for these values? If so then no issues.

Reply:
I do right above it. It is just cut off:
```
if (
      !(
        lapJSON?.dataSet?.[counter]?.["Lat."] ||
        lapJSON?.dataSet?.[counter]?.["Lon."]
      )
    ) {
      return;
    }
```

Reply:
Ohh I see, it wasn't part of the changes.